provisioner/secret.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var base = require('taskcluster-base');
+
+var Secret = base.Entity.configure({
+  version: 1,
+
+  partitionKey: base.Entity.keys.StringKey('token'),
+  rowKey: base.Entity.keys.ConstantKey('secret'),
+
+  properties: {
+    token: base.Entity.types.String,
+    expiration: base.Entity.types.Date,
+    workerType: base.Entity.types.String,
+    secrets: base.Entity.types.JSON,
```

Comment:
nit, but considering the name of the property in the response maybe this should be called `data` instead...

Note, I think this is fine.. In some ways it might be interesting to add `created`, `instanceType`, `region` and `availabilityZone` here... If nothing else then just for recording statistics...